drivers/lora/sx126x_standalone.c

```diff
@@ -56,26 +59,24 @@ static void sx126x_dio1_irq_callback(const struct device *dev,
 	struct sx126x_data *dev_data = CONTAINER_OF(cb, struct sx126x_data,
 						    dio1_irq_callback);
 
-	if (pins & BIT(sx126x_gpio_dio1.pin)) {
-		k_work_submit(&dev_data->dio1_irq_work);
-	}
```

Comment:
No need for this sanity check?

Reply:
No, the callback is only added to the DIO1 pin, so there is no other reason this would be called